### PR TITLE
Member request

### PR DESCRIPTION
**Pull request by aurexav**
Hi, I'm Xavier Lau. 
I am a core developer from Darwinia Network, having worked on Substrate and Polkadot ecosystem for the last three years.

I think I should be in the rank range of 2~3.

---

I'm:
- [top 0.82% this quarter](https://stackexchange.com/leagues/710/quarter/substrate/2022-07-01/251#251) on the [substrate.stackexchange](https://substrate.stackexchange.com/).
- the author of [array-bytes](https://github.com/hack-ink/array-bytes), and it's so proud to be a dependency of Substrate. [PR](https://github.com/paritytech/substrate/pull/12190)
- the author of [subalfred](https://github.com/hack-ink/subalfred). Its goal is to be an all-in-one Substrate development toolbox. And it gets some stars from the parity core developers. (thanks, guys)

I/my team found:
- a non-deterministic behavior on Substrate. Due to security issues, @gavofyork suggests I delete the detail. So the [issue](https://github.com/paritytech/substrate/issues/6752) is empty.
- some weird code during the Polkadot code review and reported to @kianenigma. [PR](https://github.com/paritytech/polkadot/pull/2562)
- the MMR leaves can be overwritten by the forks and reported to @tomusdrw. [Issue](https://github.com/paritytech/grandpa-bridge-gadget/issues/336)

My PR(s) related to the protocol:
- https://github.com/paritytech/substrate/pull/9700

Most of the time, I'm doing the code review. 
A little bit of shame, I don't have a lot of PRs contributions.

---

BTW, I format the markdown and sort the members by their names.

### CODE REVIEW

**Comment by rphmeier** [comment]
Xavier,

Thanks for the request. You're a longstanding member of the community who I recognize as a major contributor to stackexchange and in developer chats. Still, the Fellowship at the moment is constrained to _core protocol contributions_, and in that respect I do not see enough of a record of contribution to merit III or IV Dan. I would suggest applying for I or II Dan and to re-evaluate after some deeper protocol contributions.

Quoting from III Dan requirements:

> As a foundational grade, a Fellow represents both a ceiling and a floor: becoming
a Fellow indicates the individual has enough knowledge and skill to build
substantial protocol components (i.e. all but the major protocol components,
e.g. a pallet or 2-10,000 line crate) alone and with high expectations that they
will be completed correctly and to a high standard

**Comment by acgxv** [comment]
> Xavier,
> 
> Thanks for the request. You're a longstanding member of the community who I recognize as a major contributor to stackexchange and in developer chats. Still, the Fellowship at the moment is constrained to _core protocol contributions_, and in that respect I do not see enough of a record of contribution to merit III or IV Dan. I would suggest applying for I or II Dan and to re-evaluate after some deeper protocol contributions.
> 
> Quoting from III Dan requirements:
> 
> > As a foundational grade, a Fellow represents both a ceiling and a floor: becoming
> > a Fellow indicates the individual has enough knowledge and skill to build
> > substantial protocol components (i.e. all but the major protocol components,
> > e.g. a pallet or 2-10,000 line crate) alone and with high expectations that they
> > will be completed correctly and to a high standard

Updated.

File: README.md [hunk 1]
```diff
@@ -2,21 +2,22 @@
...
-| `shawntabrizi` | `EGVQCe73TpFyAZx5uKfE1222XfkT3BSKozjgcqzLBnc5eYo` | 4 |
-| `kianenigma` | `HL8bEp8YicBdrUmJocCAWVLKUaR2dd1y6jnD934pbre3un1` | 4 | 
-| `ordian` | `GAWwkmjbLhM5pnAVbdZEkwd3QjHE7kaxJSCi3Ec91Q3QSDW` | 3 |
-| `joepetrowski` | `GcDZZCVPwkPqoWxx8vfLb4Yfpz9yQ1f4XEyqngSH8ygsL9p` | 2 |
-| `KiChjang` | `GA3yPifemubFga7sTSFtLY2KFFiSRp6Bb8w31FS4xqgAvCz` | 2 |
-| `shaunxw` | `HxhDbS3grLurk1dhDgPiuDaRowHY1xHCU8Vu8on3fdg85tx` | 2 |
-| `ggwpez` | `Ea6jhP5gF4r7NqhkEoAXJDgSgYpNQNaTYU6gPsrEGfctaKR` | 1 |
-| `EclesioMeloJunior` | `CbCmCwFkfFkQo7bQtVczYg7sJ3oue6Ez2Z4RMGR8gi8deRk` | 1 |
-| `4meta5` | `FJq9JpA9P7EXbmfsN9YiewJaDbQyL6vQyksGtJvzfbn6zf8` | 1 |
+| Github Username     | Kusama Account ID                                 | Rank |
```
Comment (jacogr) [inline README.md]: Sorry to be a pain on this. Could we leave the re-formatting out of this PR - it makes merging in main (things go in quite often there) and checking the actual changes quite laborious.

(I do understand that "beauty is in the eye of the beer-holder", but specifically for my checks here, the above changed-to-all creates a bit of chaos)

TL;DR I think the formatting changes could possibly pass as a "stand-alone" PR (with some discussion around it, i.e. GH output won't be affected), but here would leave it out in this PR.

**Review by jacogr — DISMISSED** [review]

**Review by gavofyork — APPROVED** [review]

**Review by gavofyork — CHANGES_REQUESTED** [review]
Mis-read and thought this was an application for I Dan - that I would back happily.

**Comment by jacogr** [comment]
I'm a huge fan of the work I have been seeing, especially around the (mostly) thankless efforts on SE. 

Having said that, I'm also on agreement on a I Dan request.

Based on the II Dan requirements, I do feel we are missing the "assisted in implementing a major component" part of the requirement.

**Review by jacogr — APPROVED** [review]

**Review by gavofyork — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

**Review by andresilva — APPROVED** [review]

**Comment by kianenigma** [comment]
Late to the decision, but indeed wanted to appreciate numerous valuable and insightful bug reports that I have received from @AurevoirXavier over the years 👍
